yapapi/engine.py

```diff
@@ -125,6 +125,10 @@ def __init__(
         self._wrapped_consumers: List[AsyncWrapper] = []
 
         self._strategy = strategy
+        strategy.set_valid_prop_value_ranges(
+            {DEBIT_NOTE_ACCEPTANCE_TIMEOUT_PROP: (DEBIT_NOTE_MIN_TIMEOUT, None)}
+        )
```

Comment:
This can be also a part of the default strategy or a wrapping strategy — @shadeofblue, please share your opinion.

Reply:
Wouldn't this just overwrite the data passed to `StrategySupportingMidAgreementPayments`?